### PR TITLE
Implement support for notes

### PR DESCRIPTION
**Pull request by falko17**
This makes it possible to use notes in Bitwarden entries.[^1]

Specifically, users are able to:
* view notes in the `alt+m` menu
* use `alt+n` to copy the notes to the clipboard
* use `alt+5` to autotype the notes
* specify `notes` as a target to the `-t` parameter

Let me know if there's anything I should change here or if this is fine to merge.

[^1]: Notes are not covered by the rofi-rbw's custom fields feature because `rbw` puts them outside the `data` array in its output, making it impossible to retrieve them via rofi-rbw without the changes in this PR.

### CODE REVIEW

**Comment by fdw** [comment]
Wow, that looks like a very thorough PR! Thank you very much! 🙏

But I have one question: How do you use notes? Does autotyping them actually make sense, or should `rofi-rbw` rather just show them?

File: src/rofi_rbw/rofi_rbw.py [hunk 1]
```diff
@@ -104,9 +104,11 @@ def __parse_arguments(self) -> argparse.Namespace:
                     "Alt+2:type:username",
                     "Alt+3:type:password",
                     "Alt+4:type:totp",
+                    "Alt+5:type:notes",
                     "Alt+c:copy:password",
                     "Alt+u:copy:username",
                     "Alt+t:copy:totp",
+                    "Alt+n:copy:notes",
```
Comment (fdw) [inline src/rofi_rbw/rofi_rbw.py:111]: I'm not sure I'd add this shortcut by default. The info line is already long, so I'd rather leave it out of the default configuration. Interested users can still add it manually.

What do you think?

**Comment by falko17** [comment]
Thank you for the quick review!

> How do you use notes? Does autotyping them actually make sense, or should `rofi-rbw` rather just show them?

Most of the time, I use notes for additional information related to an account, like API keys. I don't know how other people use the notes field though, and a more common solution might be to create separate Bitwarden entries for things like API keys, so I agree that autotyping rarely makes sense, and I've removed the shortcuts.

**Comment by fdw** [comment]
I'm using custom fields for these things, so, for example, I might have a field `API-Key` on some entry. I find that easier and cleaner than a notes field.

But, in any case, it doesn't hurt to add support for notes. In short: Thank you 🙂

**Pull request merged**